pkg/klusterlet/clusterinfo/controller_test.go

```diff
@@ -220,7 +213,17 @@ func TestFailedClusterInfoReconcile(t *testing.T) {
 		t.Errorf("failed get updated clusterinfo ")
 	}
 
-	if meta.IsStatusConditionTrue(updatedClusterInfo.Status.Conditions, clusterv1beta1.ManagedClusterInfoSynced) {
+	if !meta.IsStatusConditionTrue(updatedClusterInfo.Status.Conditions, clusterv1beta1.ManagedClusterInfoSynced) {
```

Comment:
It is odd we change `meta.IsStatusConditionTrue` to `!meta.IsStatusConditionTrue` to pass the test?

Reply:
the `TestFailedClusterInfoReconcile` is to test failed synced case. so `!meta.IsStatusConditionTrue` is expected.
the previous is incorrect.